server/views/pages/prisonerList/index.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "../../partials/layout.njk" %}
+
+{% set pageId = 'prisoner-list-page' %}
+{% set pageTitle = "Manage learning and work progress" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Digital Prison Services",
+        href: dpsUrl
+      }
+    ],
+    classes: 'govuk-!-display-none-print'
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l">Manage learning and work progress</h1>
+
+      {% include './partials/searchBox.njk' %}
```

Comment:
I've implemented the Search box markup and styling, as per the design on the current prisoner list.

However... I think there are a few minor issues with it, such as spacing and HTML validation errors. I also think we should see if there is a common pattern for this and used the shared markup for it. One example is on the DPS homepage.